media/css/mozorg/home/fundraiser.scss

```diff
@@ -63,7 +63,6 @@ $image-path: '/media/protocol/img';
 .c-fundraiser-options {
     label {
         @include bidi(((margin, 0 $spacing-sm $spacing-md 0, 0 0 $spacing-md $spacing-sm),));
-        @include open-sans;
```

Comment:
Unrelated to this PR, just a teeny fix since Protocol 7 landed and I figured I'd sneak it in here rather than do another PR.